siege_utilities/reporting/analytics/polling_analyzer.py

```
@@ -371,107 +238,108 @@ def create_polling_summary(self,
                     f"{performer['name']} leads {data_type} with {performer['value']:,} {metric} "
                     f"({performer['percentage']:.1f}%)"
                 )
-            
+
             summary_parts.append("Cross-dimensional analysis reveals patterns in distribution and performance.")
-            
+
             return " ".join(summary_parts)
-            
+
         except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)